dspace/config/emails/request_item.granted

```diff
@@ -0,0 +1,26 @@
+## Sent to the person requesting a copy of a restricted document when the
+## request is granted.
+##
+## Parameters:
+##  {0} name of the requestor
+##  {1} unique ID of the requested Item
+##  {2} title of the requested Item
+##  {3} name of the grantor
+##  {4} email address of the grantor
```

Comment:
Suggestion:  continue to make this available for sites that want it, but don't use it in the stock message.

Reply:
@mwoodiupui : That's fine with me.  I'm OK with it existing, but I think it shouldn't be used by default (as I'm worried this would be an issue with GDPR and similar regulations). 

In that case, I'd recommend adding a note into the template then that the email address is currently **unused**, but is available if sites want to include the email of the grantor in the response.